### PR TITLE
Release 0.37

### PR DESCRIPTION
**Pull request by parkouss**
Do not merge this yet!

Some documentation for the future mozregression 0.37 release.

### CODE REVIEW

**Comment by parkouss** [comment]
@wlach, could you review this ?

File: _posts/2015-06-07-0.37-release.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title:  "Release 0.37"
+author: Julien Pagès
+---
+
+I'm excited to announce that mozregression now support **using taskcluster**
+**for inbound bisection** with this release!
+
+This is the major improvement, and this speed up inbound hunting (Yeah!).
+And mozregression made a first step to handle the migration from [ftp to s3]!
```
Comment (wlach) [inline _posts/2015-06-07-0.37-release.md]: Would probably reword this paragraph to something like:

"This considerably speeds up the process of downloading builds from integration branches (like mozilla-inbound) and also supports work by Mozilla Release Engineering to migrate build storage from ftp to s3."

File: _posts/2015-06-07-0.37-release.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title:  "Release 0.37"
+author: Julien Pagès
+---
+
+I'm excited to announce that mozregression now support **using taskcluster**
```
Comment (wlach) [inline _posts/2015-06-07-0.37-release.md]: now support -> now supports

File: documentation/usage.md [hunk 3]
```diff
@@ -70,6 +70,13 @@ Some useful command line usage.
 
        mozregression --command 'test-command {binary}'
 
+- Disable background download of next builds
+
+  Downloading next builds in background (while you are testing one build) is generally good as it
+  speed up the overall process. Still you can disable this with:
```
Comment (wlach) [inline documentation/usage.md]: it speed up the overall process -> it improves performance

File: documentation/usage.md [hunk 4]
```diff
@@ -80,6 +87,13 @@ Some useful command line usage.
 
        mozregression --http-cache-dir cache
 
+- keep downloading files in background
+
+  When download in background is enabled (the default), the background download that
+  is not used for testing is canceled. If you use the persist option and want to force
+  the full download of these files (to be able to use them later), you can use:
```
Comment (wlach) [inline documentation/usage.md]: , you can use -> , then you can keep these files by specifying a background download policy of keep:

**Pull request merged**